common/repo/GitRepo.java

```diff
@@ -294,8 +298,22 @@ public boolean merge(String branch, boolean remote) {
 
   @Override
   public boolean isMerged(String branch) {
-    // TODO: Implement method
-    throw new UnsupportedOperationException("Not implemented");
+    final String initialBranch = currentBranch();
+    List<String> mergedBranches = new ArrayList<>();
+    try {
+      switchToMasterBranch();
```

Comment:
Are you sure we need to switch to `master` for this to work?
https://stackoverflow.com/questions/226976/how-can-i-know-in-git-if-a-branch-has-been-already-merged-into-master

Also, there's a distinction between `git branch --merged master` and `git branch --merged`.
@vmax Which one should we use?

Reply:
@vmax Can you help out here?

Reply:
Added `TODO` for this test.